algorithms/src/snark/marlin/marlin.rs

```diff
@@ -88,10 +88,12 @@ impl<E: PairingEngine, FS: AlgebraicSponge<E::Fq, 2>, MM: MarlinMode> MarlinSNAR
 
             // Marlin only needs degree 2 random polynomials.
             let supported_hiding_bound = 1;
+            let supported_lagrange_sizes =
```

Comment:
We actually don't use the Lagrange basis SRS anymore, so we can set this to empty

Reply:
Done: 9022329a3
As I note in the TODO: shall I make a future PR removing it from the `CommitterKey`?